src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const inputs = document.querySelectorAll('input');
+const inputClass = 'field-label';
+const makeCapitalizes = (word = '') => {
+  if (word.length > 0) {
+    return (
+      word.charAt(0).toUpperCase() + word.slice(1)
+    );
+  }
+
+  return word;
+};
+
+inputs.forEach(input => {
+  const inputId = input.id;
+  let inputName = input.name;
```

Comment:
Also, you can use destructuring 
```suggestion
  const { id, name } = input;
  let inputName = name;
```